libcontainer/nsenter/nsenter_test.go

```diff
@@ -257,3 +238,23 @@ func initWaiter(t *testing.T, r io.Reader) {
 	}
 	t.Fatal("waiting for init preliminary setup:", err)
 }
+
+func reapChild(t *testing.T, parent *os.File) {
+	t.Helper()
+	decoder := json.NewDecoder(parent)
+	decoder.DisallowUnknownFields()
+	var pid struct {
+		Pid2 int `json:"stage2_pid"`
+		Pid1 int `json:"stage1_pid"`
+	}
+	if err := decoder.Decode(&pid); err != nil {
+		t.Fatal(err)
+	}
+	if pid.Pid1 == 0 || pid.Pid2 == 0 {
```

Comment:
Does this print the pids as `0 0`? (wondering if `%+v` would be useful here)

Reply:
This will print pids as `{0 0}` (I guess the order matches the order of elements in the `struct`).

Since this should never happen normally, I guess it doesn't matter.